packages/form-state-manager/demo/index.tsx

```diff
@@ -16,7 +16,7 @@ const App = () => {
       <FormStateManager onSubmit={console.log}>
         {({ handleSubmit, ...state }) => {
           return (
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit} name="field-2">
```

Comment:
Do you think it's feasible to add some generated name as a prop from `FormStateManager`? That way we can always have some name?

We can go even further and try to inject it into the first form of the child element. I know it may seem a bit too much but this could realle be an annoying bug.

Reply:
I wouldn’t do this, I don’t think it’s a huge bug - the worst thing that can happen is that after a failed submit you will be focused on a wrong input under these circumstances:

1. Having at least two forms with two inputs of the same name
2. Having the correct form as the first in dom
3. Have an error on the field with the same name
4. Do not disable buttons on invalid

With clear instructions on documenation it is simple to fix it. (And I think that even junior skilled developers will understand why it’s happening, as they see that the focused field has the same name.)

Reply:
To be clear, you don’t need to put a name there. It’s not required and it still works well. In the demo I just did it for testung purposes (to see what happens when a form and field has the same name)

Reply:
All right. We can address it if it becomes a problem.